test/testcafe/spec/EnrollAuthenticatorPhoneView_spec.js

```diff
@@ -62,7 +62,7 @@ test
     await t.expect(pageSubtitle).contains('your phone');
     await t.expect(pageSubtitle).contains('Enter the code below to verify.');
 
-    await t.expect(await enrollPhonePageObject.signoutLinkExists()).notOk();
+    await t.expect(await enrollPhonePageObject.signoutLinkExists()).ok();
```

Comment:
Another occurrence, not sure if this was intended..

@vincentvoong-okta - So this is the screen in Enroll Phone flow where we prompt the user to enter the code. With this change, we will have a `Back to sign in` link in the footer, is this ok?